.flake8

```diff
@@ -14,6 +14,9 @@ per-file-ignores =
     tests/*:S101
     # I900: Requirements for examples shouldn't be included
     examples/*:I900
+    # I900: flake8-requirements doesn't see aioredis in the requirements file cause it is extra dependency
+    aiohttp_session/redis_storage.py:I900
```

Comment:
I suspect it should be added to the requirements file for our testing. Though aioredis is archived, so we should also migrate to redis/valkey now.